src/Core/Injector/Injectable.php

```diff
@@ -20,7 +20,6 @@ trait Injectable
      *    $list = DataList::create(SiteTree::class);
      *    $list = SiteTree::get();
      *
-     * @param array $args
```

Comment:
I believe the correct syntax is: `@param mixed $args,...`

Reply:
Spot on. Thanks for that!